### PR TITLE
Docs For GEF Command

### LINKED ISSUE #643: Missing documentation for `gef` command
Reported by @khaoticdude on Discord

### PR DESCRIPTION
**Pull request by hoefler02**
### Description/Motivation/Screenshots ###
This PR adds docs for the `gef` command and its subcommands (as in issue #643).

### How Has This Been Tested? ###

N/A (docs)

### Checklist ###

- [x] My PR was done against the `dev` branch, not `master`.
- [x] My code follows the code style of this project.
- [x] My change includes a change to the documentation, if required.
- [x] My change adds tests as appropriate.
- [x] I have read and agree to the **CONTRIBUTING** document.

Fixes #643

### CODE REVIEW

File: docs/commands/gef.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+context.grow_stack_down (bool) = False
+context.ignore_registers (str) = ""
+context.layout (str) = "-code -stack"
+context.libc_args (bool) = False
+
+[...snip...]
+
+```
+
+You can use `gef config [setting] [value]` to set a setting for the current setting (see example below).
```
Comment (Grazfather) [inline docs/commands/gef.md:56]: Also `gef config [setting]` (or a prefix) to filter/view those settings.

File: docs/commands/gef.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+```
+gef➤  gef config theme.address_stack "blue"
```
Comment (Grazfather) [inline docs/commands/gef.md:59]: no quotes needed

File: docs/commands/gef.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+```
+gef➤  gef missing
+[*] Command `set-permission` is missing, reason  →  Missing `keystone-engine` package, install with: `pip install keystone-engine`.
+[*] Command `assemble` is missing, reason  →  Missing `keystone-engine` package for Python, install with: `pip install keystone-engine`.
+
+[...snip...]
+
+```
+
+As it says in the above output, the issues should be resolved by installing the missing package(s) using pip.
```
Comment (Grazfather) [inline docs/commands/gef.md:34]: Can you use your editor's line wrap on these? `gq<left arrow>` on vim.

**Review by Grazfather — APPROVED** [review]
Looks great!

**Review by hugsy — APPROVED** [review]

**Comment by hugsy** [comment]
Fixed #643

**Pull request merged**